incubator/twitter/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "kubeless-python-simple",
+  "version": "0.0.1",
+  "description": "This example demonstrates how to setup a simple Python function with Kubeless",
+  "dependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
can you try to deploy the function with serverless and check with Andres what works or does not ?thanks